public/cli/lib/commands/ffmpeg.ts

```diff
@@ -35,32 +34,29 @@ export async function combineFrames(settings: FfmpegSettings): Promise<void> {
   }
 }
 
-export async function combineAudioIfNecessary(audioFiles: string[]): Promise<string> {
-  return new Promise<string>(async (resolve, reject) => {
-    // if we have a directory, read the files in the directory
-    if (audioFiles.length > 1) {
-      // Separate mp3 and wav files
-      const mp3Files = audioFiles.filter((f: string) => f.endsWith('.mp3'));
-      const wavFiles = audioFiles.filter((f: string) => f.endsWith('.wav'));
-      // If this folder contains wav and mp3 files, then throw error
-      if (mp3Files.length > 0 && wavFiles.length > 0) {
-        reject(new Error('Conflicting audio types'));
-      }
-      // if we have wav files, then we merge them into one file
-      // and return the combined file path
-      else if (wavFiles.length > 0) {
-        resolve(await mergeWavFiles(wavFiles));
-      }
-      // if we have mp3 files, return the glob format with .mp3 files
-      else if (mp3Files.length > 0) {
-        resolve(getGlobFormat(mp3Files));
-      }
-    }
+export async function combineAudioIfNecessary(audioFiles: string[]): Promise<string | undefined> {
```

Comment:
I assume that the return type of `Promise<string | undefined>` is for the error cases e.g.
`throw new Error('Unsupported audio types');`
where a string isn't actually returned?

Reply:
Yes, that is right.